modules/apps/object/object-test/src/testFunctional/paths/Picklist.path

```diff
@@ -67,7 +67,7 @@
 	</tr>
 	<tr>
 		<td>ADD_PICKLIST_ITEM_NAME</td>
-		<td>//input[@id='locale' and @value='${key_itemName}']/ancestor::div[@class='modal-body']//input[@class='form-control' and @value='${key_itemKey}']</td>
+		<td>//input[@id='locale' and @class='form-control']</td>
```

Comment:
I think this path is too generic, meaning, if this page suffers any alterations (example: add another input or else), it might break. So it fixes the problem now, but we might have another fix in the future.
My suggestion would be to use an already created PATH, from the "generic" paths we have. You could change the path `Modal#INPUT_WITH_LABEL` to the code bellow:

> `//*[contains(@class,'modal-body')]//label[text()='${key_fieldLabel}']/..//input`

This way, we can use this generic path (that is used across testcases), but that uses the _label_ from the input entry (not getting too generic)